test/Account/unlock.test.ts

```diff
@@ -0,0 +1,63 @@
+import * as sinon from 'sinon';
+import { assert } from 'chai';
+
+import Account from '../../src/Account';
+import SpyAssert from '../utils/SpyAssert';
+
+const Web3 = require('web3');
+
+describe('Unlock', () => {
+  let web3: any;
```

Comment:
```suggestion
  let web3: Web3;
```

Reply:
We are downgrading version of web to beta 37 which does not have type script support for all types. We are thus removing using Web3 types in our code